src/cmd-buildupload

```diff
@@ -109,6 +117,12 @@ def s3_upload_build(args, builddir, bucket, prefix):
             s3_path = f'{prefix}/{nogz}'
             set_content_disposition = True
 
+        # Mark artifacts that we don't want to upload as already uploaded
+        # so they'll get ignored later.
+        if imgname not in upload_artifacts:
```

Comment:
I guess we could just do

```suggestion
        if len(args.artifact) > 0 and imgname not in args.artifact:
```

and drop `upload_artifacts`. I think we use that pattern in a bunch of places already.

Reply:
done